cmd/build-disk.go

```diff
@@ -105,7 +105,8 @@ func NewBuildDisk(root *cobra.Command, addCheckRoot bool) *cobra.Command {
 	c.Flags().Bool("expandable", false, "Creates an expandable image including only the recovery image")
 	c.Flags().Bool("unprivileged", false, "Makes a build runnable within a non-privileged container, avoids mounting filesystems (experimental)")
 	c.Flags().VarP(imgType, "type", "t", "Type of image to create")
-	c.Flags().StringSliceP("cloud-init", "c", []string{}, "Cloud-init config files")
+	c.Flags().StringSliceP("cloud-init", "c", []string{}, "Cloud-init config files to include in disk")
+	c.Flags().StringSlice("build-hooks", []string{}, "Paths to cloud-init config files to run during build")
```

Comment:
This rename is not enough, it should be adapted within the `pkg/types/v1/config.go` at `DiskSpec` definition too.

Note for later.  I am also wondering if shouldn't we make this change consistent to all commands and then have `install-hooks`, `upgrade-hooks`, `reset-hooks` flags.

Reply:
Yep, I brought back the `cloud-init-paths` for now since it is needed for this PR to produce a bootable rpi disk.

I also think we should rework the hooks in a follow up PR :+1: